sendfile/__init__.py

```diff
@@ -1,7 +1,13 @@
-VERSION = (0, 3, 10)
+from future import standard_library
+standard_library.install_aliases()
+from builtins import map
+VERSION = (0, 3, 12, 1)
 __version__ = '.'.join(map(str, VERSION))
 
-import os.path
+import os
+import contextlib
+import urllib.request
```

Comment:
you're going to want to take this and urlparse below and put them inside the `def sendfile` block.